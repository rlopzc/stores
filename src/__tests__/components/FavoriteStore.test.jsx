import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import FavoriteStore from '../../components/FavoriteStores';

describe('FavoriteStore', () => {
  const store = {position: {lat: 1, lng: 2}};
  const favorites = [store, store, store];

  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<FavoriteStore />, div);

    const component = renderer.create(<FavoriteStore />);
    expect(component).toMatchSnapshot();
  });

  it('renders favorite stores', () => {
    const component = renderer.create(<FavoriteStore stores={favorites} />);
    expect(component).toMatchSnapshot();
  });
});
