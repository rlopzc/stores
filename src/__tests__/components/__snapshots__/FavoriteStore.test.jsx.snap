// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FavoriteStore renders favorite stores 1`] = `
<div
  className="panel panel-default height-100 scrollable"
>
  <div
    className="panel-heading"
  >
    <h3
      className="panel-title"
    >
      My Favorite stores
    </h3>
  </div>
  <div
    className="panel-body"
  >
    <ul>
      <li />
      <li />
      <li />
    </ul>
  </div>
</div>
`;

exports[`FavoriteStore renders without crashing 1`] = `
<div
  className="panel panel-default height-100 scrollable"
>
  <div
    className="panel-heading"
  >
    <h3
      className="panel-title"
    >
      My Favorite stores
    </h3>
  </div>
  <div
    className="panel-body"
  >
    <p>
      No favorite stores
    </p>
  </div>
</div>
`;
